onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperator.cpp

```diff
@@ -635,6 +635,40 @@ namespace Dml
             ));
     }
 
+    void DmlOperator::ConvertNHWCToNCHW(
+        const uint32_t& dimCount, 
+        const gsl::span<const uint32_t> nhwcSizes,
+        std::vector<uint32_t>& nchwSizes,
+        std::vector<uint32_t>& nchwInputStrides)
+    {
+        int i = 0;
+        const uint32_t inputBatch = nhwcSizes[ i++ ];
```

Comment:
>nhwcSizes[ i++ ] [](http://example.com/codeflow?start=36&length=16)

`nhwcSizes[i++]`
nit - extra spaces inside braces inconsistent with elsewhere (e.g. `descs[i] = m_outputTensorDescs[i].GetDmlDesc();` above)

Reply:
ack, updating in next commit